backend/src/api/game/play.rs

```diff
@@ -0,0 +1,49 @@
+use crate::api::game::{game_request::GameRequest, game_response::GameResponse};
+use crate::server_error::ServerError;
+use actix_web::web::{self, Json, Path};
+use hive_lib::{
+    game_error::GameError, history::History, piece::Piece, position::Position, state::State,
+};
+
+fn get_game_state_from_db(_game_id: u64) -> Result<State, GameError> {
```

Comment:
imo functions like this should live in a `Game` struct in `backend/src/model/game.rs`, while the `src/api` files just have simple handlers that use the models

Reply:
I moved the two files into `model` but will not change them too much and do that when I attach the DB in the next PR this one is too big for my liking already